hadoop-ozone/common/src/main/java/org/apache/hadoop/ozone/snapshot/SnapshotDiffReport.java

```diff
@@ -199,14 +208,20 @@ public List<DiffReportEntry> getDiffList() {
   @Override
   public String toString() {
     StringBuilder str = new StringBuilder();
-    String from = "snapshot " + fromSnapshot;
-    String to = "snapshot " + toSnapshot;
-    str.append("Difference between ").append(from).append(" and ").append(to)
-        .append(":")
+    str.append("Difference between snapshot: ")
+        .append(fromSnapshot)
+        .append(" and snapshot: ")
+        .append(toSnapshot)
+        .append(",")
```

Comment:
Is this comma necessary?

Reply:
I felt it is better to have it. Removed it.